schedule/qam/common/create_hdd_transactional_server.yaml

```diff
@@ -11,16 +11,17 @@ conditional_schedule:
     kernel_cmd_parameters:
         BACKEND:
             pvm_hmc:
-            - installation/edit_optional_kernel_cmd_parameters
+                - installation/edit_optional_kernel_cmd_parameters
```

Comment:
apparently we don't run for anything different than x86_64, so we could remove `conditional_schedule`.